### DISCUSSION TITLE
v2.4 Grafana

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#12189] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

256

### Storage for /nsm

256

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Has the InfluxDB data schema changed from SOv2.3 to SOv2.4? Finding InfluxDB isn't meeting the mark as its sluggish and slow comparative to it's predecessor, Grafana. Would installing the _so-grafana_ container from SOv2.3 work w/ SOv2.4?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment, marked as answer]
We went to Influx 2 in 2.4 so the old container will not work. You can still connect an external grafana to it and build your own dashboards. In 2.4.40 we are adding metrics to the grid screen to lower the dependence on having to go into influx at all.
